assets/js/script.js

```diff
@@ -19,6 +21,32 @@ function checkForm(form) {
   return false;
 }
 
+/* localStorage */
+let formData = { name: '', email: '', message: '' };
+
+const onChangeHandler = (event) => {
+  switch (event.name) {
+    case 'name':
+      formData = { ...formData, name: event.value };
+      break;
+    case 'email':
+      formData = { ...formData, email: event.value };
+      break;
+    case 'message':
+      formData = { ...formData, message: event.value };
+      break;
+    default:
+      break;
+  }
+  localStorage.setItem('data', JSON.stringify(formData));
+};
+const reloadBrowser = JSON.parse(localStorage.getItem('data'));
+if (reloadBrowser) {
+  formName.value = reloadBrowser.name;
+  email.value = reloadBrowser.email;
+  message.value = reloadBrowser.message;
+}
```

Comment:
**Changes Required**
- [ ] Good effort on implementing most of the functional requirements :+1:  However, when I submit the form, the `localStorage` is cleared which is not the desired behavior. Let us try to make these changes.

Reply:
As we discussed on the zoom call, the problem hasn't been solved yet. You will need to fix the bug in order to move forward. 💪🏾